net/lnet/ulnet_core.c

```diff
@@ -1050,34 +1035,26 @@ M0_INTERNAL int nlx_core_nidstr_encode(struct nlx_core_domain *cd,
    This call is not protected by any mutex. It relies on the
    application to not invoke it during domain finalization.
  */
-M0_INTERNAL int nlx_core_nidstrs_get(struct nlx_core_domain *cd,
-				     char *const **nidary)
+M0_INTERNAL int nlx_core_nidstrs_get(struct nlx_core_domain   *cd,
```

Comment:
Do we still need domain as argument, as i don't see it anywhere being used ?

Reply:
I did this just to have `nlx_core_domain` as an argument for an `nlx_core_*` function.